backend/infrahub/graphql/queries/diff/tree.py

```diff
@@ -166,6 +180,43 @@ async def to_diff_tree(
             num_conflicts=enriched_diff_root.num_conflicts,
         )
 
+    def _get_parent_info(
+        self, diff_node: EnrichedDiffNode, graphql_context: GraphqlContext | None = None
+    ) -> ParentNodeInfo | None:
+        for r in diff_node.relationships:
+            for n in r.nodes:
+                relationship_name: str = "undefined"
```

Comment:
I know you've only moved code around here, just highlighting. Might we have any weird scenarios if we have a schema where the relationship name is "undefined"? I'm assuming this variable will almost always be replaced.